install/installer/pkg/components/agent-smith/configmap.go

```diff
@@ -21,25 +19,20 @@ func configmap(ctx *common.RenderContext) ([]runtime.Object, error) {
 	ascfg := config.ServiceConfig{
 		PProfAddr:      fmt.Sprintf("localhost:%d", PProfPort),
 		PrometheusAddr: fmt.Sprintf("localhost:%d", PrometheusPort),
+		Namespace:      ctx.Namespace,
 		Config: config.Config{
-			Blocklists: &config.Blocklists{
-				Very: &config.PerLevelBlocklist{
-					Signatures: []*classifier.Signature{{
-						Name:    "testtarget",
-						Domain:  classifier.DomainProcess,
-						Kind:    classifier.ObjectELFSymbols,
-						Pattern: []byte(base64.StdEncoding.EncodeToString([]byte("agentSmithTestTarget"))),
-						Regexp:  false,
-					}},
-				},
-			},
-			Kubernetes: config.Kubernetes{Enabled: true},
+			Kubernetes:          config.Kubernetes{Enabled: true},
+			KubernetesNamespace: ctx.Namespace,
 			GitpodAPI: config.GitpodAPI{
 				HostURL: fmt.Sprintf("https://%s", ctx.Config.Domain),
 			},
 		},
 	}
 
+	if ctx.Config.AgentSmith != nil {
+		ascfg.Config = *ctx.Config.AgentSmith
```

Comment:
Can we overwrite `KubernetesNamespace` and `GitpodAPI` here?

Reply:
We can overwrite `KubernetesNamespace` but probably not `GitpodAPI` since it requires a token to be present as well. Unless it is already available somewhere in the installer yaml file? :thinking:

Reply:
I am sorry, I meant, to use the values from the installer as the source.